frontend/pages/admin/TeamManagementPage/TeamDetailsWrapper/UsersPage/UsersPage.tsx

```diff
@@ -76,36 +77,36 @@ const MembersPage = ({ location, router }: ITeamSubnavProps): JSX.Element => {
   const [editUserErrors, setEditUserErrors] = useState<IUserFormErrors>(
     DEFAULT_CREATE_USER_ERRORS
   );
-  const [memberIds, setMemberIds] = useState<number[]>([]);
+  const [userIds, setUserIds] = useState<number[]>([]);
 
   const toggleAddUserModal = useCallback(() => {
-    setShowAddMemberModal(!showAddMemberModal);
-  }, [showAddMemberModal, setShowAddMemberModal]);
+    setShowAddUserModal(!showAddUserModal);
+  }, [showAddUserModal, setShowAddUserModal]);
 
-  const toggleRemoveMemberModal = useCallback(
+  const toggleRemoveUserModal = useCallback(
     (user?: IUser) => {
-      setShowRemoveMemberModal(!showRemoveMemberModal);
+      setShowRemoveUserModal(!showRemoveUserModal);
       user ? setUserEditing(user) : setUserEditing(undefined);
     },
-    [showRemoveMemberModal, setShowRemoveMemberModal, setUserEditing]
+    [showRemoveUserModal, setShowRemoveUserModal, setUserEditing]
   );
 
   // API CALLS
 
   const {
-    data: members,
-    isLoading: isLoadingMembers,
-    error: loadingMembersError,
+    data: users,
+    isLoading: isLoadingUsers,
+    error: loadingUsersError,
     refetch: refetchUsers,
-  } = useQuery<IUser[], Error, IMembersTableData[]>(
+  } = useQuery<IUser[], Error, IUsersTableData[]>(
     ["users", teamIdForApi, searchString],
     () =>
       usersAPI.loadAll({ teamId: teamIdForApi, globalFilter: searchString }),
     {
       enabled: isRouteOk && !!teamIdForApi,
       select: (data: IUser[]) => generateDataSet(teamIdForApi || 0, data), // Note: `enabled` condition ensures that teamIdForApi will be defined here but TypeScript can't infer type assertion
       onSuccess: (data) => {
-        setMemberIds(data.map((member) => member.id));
+        setUserIds(data.map((user) => user.id));
```

Comment:
I don't think we need to hold user ids as state variables like this. We could just say, outside of this useQuery call, `userIds = teamUsers.map(teamUser => teamUser.id)`, to just reference `useQuery`'s state instead of having another state/setter.

Reply:
yeah it's not being modified again so I'mma pull it out. This is out of the scope of this work but I like your enthusiasm for code cleanup as you go. Hashtag 2024 goals